### PR TITLE
fix(wsl): get correct distro name

### PR DESCRIPTION
**Pull request by lewis-yeung**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines

### Description

Store the correct distro name of WSL into `omp.cache` so that in the "OS" segment, an icon of WSL distro will be rendered instead of a generic one. Related issue: #1766.

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: src/environment/unix.go [hunk 1]
```diff
@@ -65,15 +65,15 @@ func (env *ShellEnvironment) Platform() string {
...
-		platform = strings.ToLower(wsl)
-		return strings.Split(platform, "-")[0]
-	}
-	platform, _, _, _ = host.PlatformInformation()
-	if platform == "arch" {
-		// validate for Manjaro
-		lsbInfo := env.FileContent("/etc/lsb-release")
-		if strings.Contains(strings.ToLower(lsbInfo), "manjaro") {
-			platform = "manjaro"
+		platform = strings.Split(strings.ToLower(wsl), "-")[0]
```
Comment (JanDeDobbeleer) [inline src/environment/unix.go:68]: Can you do an early return here rather than use an else statement?

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: src/environment/unix.go (hunk 1 above)
Comment (lewis-yeung) [reply src/environment/unix.go:68]: @JanDeDobbeleer Updated.

**Comment by JanDeDobbeleer** [comment]
@lewis-yeung your commit message paragraph is too long ;-)

> Store the correct distro name of WSL into `omp.cache` so that in the "OS" segment, an icon of WSL distro will be rendered instead of a generic one. Related issue: #1766.
✖   footer's lines must not be longer than 100 characters [footer-max-line-length]

**Comment by lewis-yeung** [comment]
@JanDeDobbeleer Changed.

**Pull request merged**